src/Interfaces.tsx

```diff
@@ -0,0 +1,19 @@
+export interface IState {
```

Comment:
Nice job moving your interfaces into their own file.  Curious if you could explain to me what an interface is?  Is it similar to something like proptypes?